qa/crontab/teuthology-cronjobs

```diff
@@ -0,0 +1,280 @@
+## AUTOMATED CRONTAB UPDATING
+## https://code.google.com/archive/p/chkcrontab/wikis/CheckCrontab.wiki
+## https://github.com/ceph/ceph-cm-ansible/pull/391
+# chkcrontab: disable-msg=INVALID_USER
+# chkcrontab: disable-msg=USER_NOT_FOUND
+REMOTE_CRONTAB_URL="https://raw.githubusercontent.com/ceph/ceph/master/qa/crontab/teuthology-cronjobs"
```

Comment:
@djgalloway does this look right?

Reply:
No.  Use the text in this comment exactly: https://github.com/ceph/ceph/pull/21075#issuecomment-376671188